changelog/4935.feature.rst

```diff
@@ -0,0 +1,15 @@
+Add option ``use_cls_token`` to all tokenizers. If it is set to ``True``, the token ``__CLS__`` will be added to
+the end of the list of tokens.
+
+Add option ``return_sequence`` to all featurizers. By default all featurizers return a matrix of size
+(1 x feature-dimension). If the option ``return_sequence`` is set to ``True``, the corresponding featurizer will return
+a matrix of size (token-length x feature-dimension).
+
+Split featurizers into sparse and dense featurizers.
+
+Deprecate ``NGramFeaturizer``. Please use ``CountVectorsFeaturizer`` instead.
+
+To use custom features in the ``CRFEntityExtractor`` use ``text_dense_features`` instead of ``ner_features``. If
```

Comment:
should also go into a separate removal entry (prob makes sense to create an issue for this to link to). should also link to tdocs

Reply:
https://github.com/RasaHQ/rasa/issues/4957